lib/trailblazer/macro/guard.rb

```diff
@@ -1,6 +1,7 @@
 module Trailblazer::Macro
   module Policy
-    def self.Guard(proc, name: :default, &block)
+    # noinspection RubyClassMethodNamingConvention
```

Comment:
Should we disable this in main rubocop config itself ?

Reply:
Yes please, I don't see the idea behind adding all this ~crap~ confusing syntax to actual _code_.